retrofit2/src/main/java/com/linecorp/armeria/client/retrofit2/ArmeriaCallFactory.java

```diff
@@ -151,8 +152,11 @@ private static HttpResponse doCall(ArmeriaCallFactory callFactory, Request reque
                 default:
                     throw new IllegalArgumentException("Invalid HTTP method:" + request.method());
             }
-            request.headers().toMultimap().forEach(
-                    (key, values) -> headers.add(HttpHeaderNames.of(key), values));
+            final int numHeaders = request.headers().size();
```

Comment:
`request.headers()` could be extracted into a local variable?

Reply:
Fixeed